content/en/network_monitoring/devices/guide/device_profiles.md

```diff
@@ -102,44 +167,134 @@ Adding tags to tabular metrics is similar to adding global tags, with two additi
+| No Modification | The device's returned value will be used directly as the tag value.                                 |
+| Format          | This can be [mac_address][5] or [ip_address][6].                                                    |
+| Extract Value   | A regular expression used to [extract][7] the tag value from the SNMP value provided by the device. |
+| Mapping         | This is described [here][8].   
+
+[5]: https://datadoghq.dev/integrations-core/tutorials/snmp/profile-format/#format-mac_address
+[6]: https://datadoghq.dev/integrations-core/tutorials/snmp/profile-format/#format-ip_address
+[7]: https://datadoghq.dev/integrations-core/tutorials/snmp/profile-format/#extract_value
+[8]: https://datadoghq.dev/integrations-core/tutorials/snmp/profile-format/#mapping-index-to-tag-string-value
+
+{{% /collapse-content %}}
+
+{{% /tab %}}
+{{< /tabs >}}
 
 ### Apply a profile to created devices
 
-1. After you save a profile, navigate back to the [profile home page][4] and select the **Download All Profiles** option. This allows you to download the `.zip` bundle which contains the `yaml` files for the profiles you created. 
+{{< tabs >}}
+{{% tab "Automatic (recommended)" %}}
+
+After applying your configuration options to your device profile, click **Save and Sync Agents** to automatically apply this profile to all NDM agents. The configurations are applied to your devices with [Remote Configuration][14].
+
+{{< img src="/network_device_monitoring/profile_onboarding/save_sync_agents.png" alt="The Network Device profile page showing the final step to save and sync agents" style="width:100%;">}}
+
+[14]: /agent/remote_config
+
+{{% /tab %}}
+
+{{% tab "Manual" %}}
+
+1. After you save a profile as a draft, navigate back to the [profile home page][4], select the **Draft Profiles** tab, then select the **Download All Profiles** option. This allows you to download the `.zip` bundle which contains the `yaml` files for the profiles you created. 
 2. Place the `yaml` files in the [profile directory][13] on each of the relevant installed Agents.
 3. Restart the Datadog Agent.
 4. To ensure the profiles you created are accurate, confirm that NDM is receiving metrics from the matched devices as expected.
 
+{{< img src="/network_device_monitoring/profile_onboarding/profile_download.png" alt="The Network Device profile main page highlighting the Download All Profiles option" style="width:100%;">}}
+
+[4]: https://app.datadoghq.com/devices/profiles
+[13]: https://github.com/DataDog/integrations-core/tree/master/snmp/datadog_checks/snmp/data/profiles
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Inventory page
+
+Navigate to [Infrastructure > Network Devices > Configuration][1] to view the device profile [Inventory][4] page. This is where you can see a snapshot of the device profiles you created using the [Device Profile Setup](#setup) below. 
```

Comment:
The view shows also the datadog OOTB profiles (which were not created through the UI)